libs/barista-components/button/src/_button-theme.scss

```diff
@@ -87,7 +87,7 @@ a.dt-button-secondary:visited {
   }
 }
 .dt-button-secondary:hover:not([disabled]) {
-  background: $gray-100;
+  background: rgba(255, 255, 255, 0.2);
```

Comment:
@tomheller should we put this in a design token?

Reply:
I don't think it is necessary to put this into a design token. What would make sense is to create opacity (or transparency ones, whatever feels better) tokens, and use the transparentize function from sass to create the color. Could look something like:
```scss
background: transparentize($white, $opacity-low);
```

Reply:
Same pattern we use few lines above :) for 
`.dt-button-primary[disabled] {`

Will change it when transparentize(...) fits here more,
@tomheller is @white +@opacity-low just an example howto use it or we could go with that?

Reply:
@JanuszLis this was just an example of how fixed color variables from our tokens could work with transparencies. I don't think you need to go with it. I guess this is something we should tackle on a more global scale (as you mentioned this appears in other places as well). 
Furthermore there are no opacity tokens set up at this moment. 

This suggestion was more a outlook into the future and how this could be solved.

Reply:
ok now I get it ;) so leaving current changes as they are. Feel free for any more feedback or review decisions ;)